Competition2017/src/competition/subsystems/shooter/SideShooterSubsystem.java

```diff
@@ -0,0 +1,166 @@
+package competition.subsystems.shooter;
+
+import org.apache.log4j.Logger;
+
+import com.ctre.CANTalon.FeedbackDevice;
+import com.ctre.CANTalon.TalonControlMode;
+
+import edu.wpi.first.wpilibj.Timer;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.BooleanProperty;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class SideShooterSubsystem extends BaseSubsystem {
+    
+    private static Logger log = Logger.getLogger(ShooterSubsystem.class);
+    
+    private final XCANTalon masterMotor;
+    public final String side;
```

Comment:
Uh I can convert it to a enum type. I didn't do it previously bc I didn't know what a enum was